translations.txt

```diff
@@ -390,3 +390,232 @@ This will reboot the system=
+Trial has expired, see options below=
+Opens %s in new tab=
+My Servers Dashboard=
+Unraid Forums=
+Settings > Management Access • Unraidnet=Settings > Management Access • Unraid.net
+Trial key expires at %s=
+Trial expires in %s=
+Trial key expired at %s=
+Trial expired %s=
+Server up since %s=
+Uptime %s=
+My Servers=
+beta=
+Unraid API Error=
+Failed to connect to Unraid API=
+My Servers Error=
+Please wait a moment and reload the page=
+Close Details=
+Loading My Servers data=
+Displaying last known server data=
+Current server=
+Local access=
+Server Offline=
+Remote access=
+Access unavailable=
+Opens new HTTPS window to %s=
+Sign In to resolve=
+Sign In to Purchase Key=
+Purchase Key or Extend Key=
+No Keyfile=
+Thanks for supporting Unraid=Thanks for supporting Unraid!
+Lets unleash your hardware=Let's unleash your hardware!
+You are all set 👍=
+Sign in or sign up to get started=
+Trial=
+Thank you for choosing Unraid OS=
+Your Trial key includes all the functionality and device support of a Pro key=
+After your Trial has reached expiration, your server still functions normally until the next time you Stop the array or reboot your server=After your *Trial* has reached expiration, your server **still functions normally** until the next time you Stop the array or reboot your server
+At that point you may either purchase a license key or request a Trial extension=At that point you may either purchase a license key or request a *Trial* extension
+You have %s remaining on your Trial key=You have **%s** remaining on your Trial key
+Trial Expired=
+Your Trial has expired=
+To continue using Unraid OS you may purchase a license key=
+You have used all your Trial extensions=You have used all your Trial extensions.
+Alternately, you may request a Trial extension=
+Basic=
+Plus=
+Pro=
+GUID Error=
+Registration key  GUID mismatch=Registration key / GUID mismatch
+The license key file does not correspond to the USB Flash boot device Please copy the correct key file to the /boot/config directory on your USB Flash boot device=The license key file does not correspond to the USB Flash boot device. Please copy the correct key file to the **/boot/config** directory on your USB Flash boot device
+or click Replace Key to transfer your registration to this new flash device=
+Missing key file=
+It appears that your license key file is corrupted or missing The key file should be located in the /boot/config directory on your USB Flash boot device If you do not have a backup copy of your license key file you may attempt to recover your key If this was a Trial installation, you may purchase a license key=It appears that your license key file is corrupted or missing. The key file should be located in the **/boot/config** directory on your USB Flash boot device. If you do not have a backup copy of your license key file you may attempt to recover your key. If this was a Trial installation, you may purchase a license key.
```

Comment:
slashes on left

Reply:
Ahh thanks. Had just focused on the asterisks and glazed over the slashes